consumer/consumer.go

```diff
@@ -218,3 +216,73 @@ type offsetStash interface {
 type highWaterMarker interface {
 	HighWaterMarkOffset() int64
 }
+
+// Message represents a message received from Kafka.
+// When using Felice's Consumer, any Handlers that you register
+// will receive Messages as they're arguments.
+type Message struct {
+	// The Kafka topic this Message applies to.
```

Comment:
🌷 Here the first letter of the word message is in capital, below it's in lowercase. Make a choice 😉